services/app/src/pages/game/Game.jsx

```diff
@@ -53,12 +55,25 @@ function Room() {
     [users, currentUser, movedBy],
   );
 
+  useEffect(() => {
+    if (playerDeleted && playerDeleted === currentUser.id) {
+      localStorage.removeItem('user');
+      window.location.reload();
```

Comment:
how was this resolved? reload() is still there...

Reply:
It's strange, if you look at the last commit "merged files" you will see that in game.jsx there is window.location.replace('/') instead of window.location.reload()